rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -44,6 +44,8 @@ struct PlayOptions
   // Rate in Hz at which to publish to /clock.
   // 0 (or negative) means that no publisher will be created
   double clock_publish_frequency = 0.0;
+
```

Comment:
Can you add a comment here stating what `delay` is and the unit?

Reply:
I added in https://github.com/ros2/rosbag2/pull/789/commits/d10ee937e26570f29298545e446e51ebd6bfd24d.